index.js

```diff
@@ -1,14 +1,27 @@
-const { curryN } = require('crocks')
+const { curryN }         = require('crocks')
+const { flip, is, when } = require('ramda')
+
+const addReqData = (err, req) =>
```

Comment:
We should switch this param order to be `(req, err)` to match the `handle` function.